srcpkgs/fcft/template

```diff
@@ -27,6 +24,10 @@ fcft-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		vmove usr/share/doc
 		vmove usr/share/man/man3
+
+		# The license is already installed by the base package
+		rm ${PKGDESTDIR}/usr/share/doc/${sourcepkg}/LICENSE
```

Comment:
Does this need to be deleted? I don't see an issue with having it there.

Reply:
Is it not strange to have a license in the doc folder when we have a license folder? Is it not redundant to have the license in both `fcft` and `fcft-devel`?

Reply:
I agree with both those points, it's mostly that I don't know if they make it worth having one more thing in the template. But you're the maintainer for it, so it definitely makes sense for you to make the decision on that :p

Reply:
> I can't find the resolve issue button, so I can't close this thread, sorry.

No worries I managed to find it :D